lib/devos/mkFlake.nix

```diff
@@ -0,0 +1,198 @@
+{ lib, flake-utils, deploy }:
+let
+  mkFlake = { self, ... } @ args:
+      # TODO: pipe args through importers
```

Comment:
@Pacman99 I was counting on your expertise with the module system, etc. here, I did remove it, because I'm not familiar, so I can leave that out of concern for now. But it is meant to go here, for sure.

Reply:
Yup! Once we switch to module system, we should drop `mkFlake`' entirely and not specify required arguments in nix. As all arguments should be defined in the module, that way deprecating parts of api is easier. Only `self` should be included, since that is necessary to get first.

Reply:
The main use case of `mkFlake'` is the one of people not wanting to use devos' file system contracts. We definitely should not attempt to provide a template on how to use it, but people might dislike certain things about how things are imported.